### ISSUE TITLE
add faq question

### ISSUE THREAD

**Issue by xyb994** [open-learning-exchange/open-learning-exchange.github.io#3528]
### Please verify the following:

- [X] This issue is related to the [Mobile Interns First Steps](https://open-learning-exchange.github.io/#!pages/mi/mi-10-steps.md) document or [myPlanet User Manual](https://open-learning-exchange.github.io/#!pages/manual/myplanet/overview.md).
- [X] You have searched for relevant keywords in [recent open and closed issues](https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues?q=is%3Aissue+label%3Ami) to avoid duplicates.

### URL to the MDwiki page containing the problem

https://open-learning-exchange.github.io/#!pages/mi/mi-first-steps.md

### Problem

Multiple prospective interns ran into issues with staled master branch on their fork due to not carefully following the steps. We've been addressing these issue individually and could use a FAQ for it.

### Proposed solution(s)

Add content from https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/3504#issuecomment-2518261253 to Technical Questions Q4

**Comment by MelonLord8** [comment]
Perhaps we could survey the successful mobile interns and ask them if they had any confusions or questions throughout the process, and then add new entries to the FAQ page as seen fit.

**Comment by tuyisabeangelique** [comment]
This is a great idea. Adding an FAQ for addressing stale master branches will definitely reduce the repetitive effort of resolving the same issue for prospective interns. Perhaps we could also mention tools or commands like `git pull upstream master` as part of the solution?

**Comment by munishpatel** [comment]
Following the commands below can be useful while synchronizing your repository
`git fetch upstream`
`git checkout master`
`git merge upstream/master`
`git push origin master`

**Comment by Sakshamnayyar** [comment]
This is a great idea. But I do wanna mention that this is already present in the mi-github-issues.md -> 4. Synchronize Your Repository.

**Comment by Sakshamnayyar** [comment]
Hey @xyb994, if no one is working on it, can I work on this issue?

**Comment by xyb994** [comment]
> Hey @xyb994, if no one is working on it, can I work on this issue?

@Sakshamnayyar I don't believe anyone is working on this. Check this is not already in the FAQ and free to grab it :)

**Comment by ragilzakaria** [comment]
Having this added to the FAQ will be extremely helpful and save a lot of repetitive debugging on Discord. A stale master branch is a very common hurdle when interns are first learning to sync their forks. In addition to the CLI commands (using  git fetch upstream  and merging), it might also be helpful to include a quick note or screenshot of the GitHub Web UI's "Sync fork" button as an alternative method.
